runtime/ftplugin/qf.vim

```diff
@@ -14,3 +14,12 @@ let b:undo_ftplugin = "set stl<"
 
 " Display the command that produced the list in the quickfix window:
 setlocal stl=%t%{exists('w:quickfix_title')?\ '\ '.w:quickfix_title\ :\ ''}\ %=%-15(%l,%c%V%)\ %P
+
+autocmd Syntax <buffer>
```

Comment:
What happens if I open a help window, open the location list. then switch to a man buffer. Will this autocmd still run and will the syntax change from `'help'` to `'man'`?

I'm not sure because you have `silent %delete _` in the command that is run, so I think the location list window is reused. And if so, then the syntax would be `'help'` and so the if statement would evaluate to false.

I don't have a laptop for a while, so I can't test this, just making sure.